sql/parser/builtins.go

```diff
@@ -761,3 +773,73 @@ func datumToRawString(datum Datum) (string, error) {
 
 	return "", fmt.Errorf("argument type unsupported: %s", datum.Type())
 }
+
+// Round rounds x to n digits.
+// Directly taken from CPython's `double_round` in
+// cpython/Object/floatobject.c.
```

Comment:
I'm happy with either this version, the `snprintf` or no `round` support at all.

I saw that the `snprintf` is preferred. It might be more portable or can handle a wider domain of inputs? After all it depends how it is used. Maybe it's best to not provide `round` and wait until users present a case. Or mark it as experimental.